commit/plugin.go

```diff
@@ -157,7 +157,7 @@ func (p *Plugin) Observation(
 	// If there's no previous outcome (first round ever), we only observe the latest committed sequence numbers.
 	// and on the next round we use those to look for messages.
 	if outctx.PreviousOutcome == nil {
-		p.lggr.Debugw("first round ever, can't observe new messages yet")
+		p.lggr.Infow("first round ever, can't observe new messages yet")
```

Comment:
Making these info because turning on debug logs in the CL node brings an avalanche of logs that are not relevant.

Reply:
but then it will always show even without turning on debug logs, right?

Reply:
Log levels filter downards, so Debug means Debug and all levels above, Info means Info and all levels above, etc.